src/stdlib.js

```diff
@@ -11,6 +11,27 @@ export const lightscriptImports = {
   "bitwiseZeroFillRightShift": "inline",
 };
 
+export const runtimeHelpers = {
```

Comment:
Sorry, I hadn't read that comment very carefully – I did check in the babel repl that with an ie<6 target the output doesn't change